docs/operating-scylla/procedures/cluster-management/rebuild-node.rst

```diff
@@ -2,18 +2,24 @@
 Rebuild a Node After Losing the Data Volume
 ============================================
 
-When running in Scylla on EC2, it is recommended to use i3 type instances, storing the data on fast, ephemeral SSD drives.
+When running in Scylla on EC2, it is recommended to use i4i and i3en type instances, storing the data on fast, ephemeral SSD drives.
 Stopping the node and starting it for whatever reason means the data on the node is lost.
-This data loss applies not only to i3 type instances but to the i2 type as well.
+
 The good news is Scylla is a HA database, and replicas of the data are stored on additional nodes.
-This is also why it's highly recommended to use a replication factor of at least three per Data Center (for example, 1 DC, RF = 3, 2 DCs RF = 6).
+This is also why it's highly recommended to use a replication factor of at least three per Data Center, for example, ``replication = {'class': 'NetworkTopologyStrategy', 'DC1' : 3, 'DC2' : 3}``.
 
 To recover the data and rebuild the node, follow this procedure:
 
 #. Open the ``/etc/scylla/scylla.yaml`` file.
 
 #. Add, if not present, else edit, the ``replace_node_first_boot`` parameter and change it to the
    Host ID of the node before it restarted.
+
+#. If you know of dead nodes in the cluster, for example, from :doc:`nodetool status </operating-scylla/nodetool-commands/status>`, you can use :doc:`ignore_dead_nodes_for_replace </reference/configuration-parameters>` to ignore these nodes for the rebuild.
```

Comment:
Although the functionality is rebuilding the node, what actually happens here is the node replacing itself, using its previous identity (Host ID), and using the same ip address, in contrast to using `nodetool rebuild`, which is a slightly different beast, see https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/rebuild.html.
So the use of `rebuild` in this context is a bit confusing.

I think it's worth adding a short paragraph that to rebuild the node, we apply https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/replace-dead-node.html with the special case of replacing the node "in place".

Reply:
yes, word rebuild should be exchanged for replace (as we use "replace" read node)
and that reference should be on top, this doc is basically just applying that process in the case of instance stop/start (and loosing ephemeral disks)